### ISSUE TITLE
Customize colors/themes

### ISSUE THREAD

**Issue by musjj** [Canop/bacon#215]
It would be nice if you can customize the colors/themes of the TUI. Or at least make it follow the user's terminal color scheme.

**Comment by Canop** [comment]
Can you clarify the problem you encounter with some screenshots and your OS+terminal configuration ?

**Comment by musjj** [comment]
It looks like this:

![image](https://github.com/user-attachments/assets/282acb4b-363d-4b9c-9886-bf4b41b8f4bb)

The hardcoded brown and pink does not come from the colors I set for my terminal, confirmed with a color picker.

This is what my terminal colors look like, as printed by `msgcat --color=test`:

![image](https://github.com/user-attachments/assets/00725bc3-7946-4676-ac23-7e40406a97e4)

**Comment by Canop** [comment]
Ok, but what's the problem ?

**Comment by musjj** [comment]
Yes, but the problem is that bacon is using the full range of 256 colors, which is almost never controlled by the user.

By default, most terminal applications uses the basic set of 8-16 colors from here: https://gist.github.com/fnky/458719343aabd01cfb17a3a4f7296797#8-16-colors. These are the colors chosen by the user when they theme their terminal. So following this palette will automatically give the application an appearance that is consistent with the user's preferences.

**Comment by mhedgpeth** [comment]
I'd be interested in making this change as a PR. @Canop what are your thoughts on this level of configuration? I see in the code that the colors are hard coded; it seems like it would be fairly straightforward to make them configurable.

**Comment by Canop** [comment]
I'm not interested by a PR there. Doing the code isn't the hard part.

**Comment by mhedgpeth** [comment]
I'm in the process of writing a console app and terminal UI for my work, so I'd love to hear your perspective if you have another minute. What challenges do you foresee with implementing this? I.e. what do you see as the hard part? Thanks again for the quick response and the fantastic tool; today it saved me so much time and gave me perspective. It was the missing piece... :)

**Comment by mahume** [comment]
@mhedgpeth, if you ever get around to opening a PR, I'd appreciate if you post it here. I'd prob just fork this repo and rebase your changes because the colors are quite hard to see against my color theme.

**Comment by Canop** [comment]
Can you please have a look at https://github.com/Canop/bacon/pull/366 ?
